packages/compass-preferences-model/src/preferences.ts

```diff
@@ -0,0 +1,318 @@
+import storageMixin from 'storage-mixin';
+import isEmpty from 'lodash.isempty';
+import pickBy from 'lodash.pickby';
+import { promisifyAmpersandMethod } from '@mongodb-js/compass-utils';
+import type { AmpersandMethodOptions } from '@mongodb-js/compass-utils';
+import createDebug from 'debug';
+
+const debug = createDebug('mongodb-compass:models:preferences');
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Model = require('ampersand-model');
+
+export enum THEMES {
+  DARK = 'DARK',
+  LIGHT = 'LIGHT',
+  OS_THEME = 'OS_THEME',
+}
```

Comment:
```suggestion
type THEMES = 'DARK' | 'LIGHT' | 'OS_THEME';
```

as we usually tend to do today? We only use this in TS code so there shouldn’t be much need for an enum (completely optional though)

Reply:
But then it will be only a type and we won't be able to access it as a value as it is used in code right now, eg:
```ts
const { THEMES } = require('compass-preferences-model');
if (theme === THEMES.OS_THEME
    && remote.nativeTheme.shouldUseDarkColors
  ) { }
```

Reply:
Yes – didn’t realize that there’s one place in which we access it from vanilla JS code :)